### PR TITLE
github/ci: Assorted cleanups

### PR DESCRIPTION
**Pull request by phlax**
- fix signoff on version history sync
- assorted cleanups

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: .github/workflows/envoy-release.yml [hunk 1]
```diff
@@ -53,6 +53,8 @@ jobs:
       with:
         app_id: ${{ secrets.ENVOY_CI_PUBLISH_APP_ID }}
         app_key: ${{ secrets.ENVOY_CI_PUBLISH_APP_KEY }}
+        committer-name: ${{ env.COMMITTER_NAME }}
+        committer-email: ${{ env.COMMITTER_EMAIL }}
```
Comment (phlax) [inline .github/workflows/envoy-release.yml:57]: requires newer toolshed version

Comment (phlax) [reply .github/workflows/envoy-release.yml:57]: ditto below

**Comment by phlax** [comment]
this breaks in the PR but should resolve once landed - i have tested pretty extensively

**Comment by phlax** [comment]
rewipped to fix issue

**Review by wbpcode — APPROVED** [review]
LGTM. Thanks.

**Review by wbpcode — APPROVED** [review]
LGTM. Thanks

**Pull request merged**